langchain/agents/agent_toolkits/openapi/planner.py

```diff
@@ -185,6 +190,7 @@ def create_openapi_agent(
     api_spec: ReducedOpenAPISpec,
     requests_wrapper: RequestsWrapper,
     llm: BaseLanguageModel,
+    shared_memory: ReadOnlySharedMemory,
```

Comment:
lets maybe make this optional, i dont think always needed

Reply:
Good point; I'll update

Reply:
Updated to optional in https://github.com/hwchase17/langchain/pull/2475/commits/678a40d1d8d1ee40219ac7474f81b7f2c9ecd3b4